Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/SamlConstants.java

```diff
@@ -100,6 +100,8 @@
     public static final String USER_ORG_ID_ATTR = "urn:oasis:names:tc:xspa:1.0:subject:organization-id";
     public static final String HOME_COM_ID_ATTR = "urn:nhin:names:saml:homeCommunityId";
     public static final String USER_ROLE_ATTR = "urn:oasis:names:tc:xacml:2.0:subject:role";
+    public static final String USER_SYST_ATT = "2.16.840.1.113883.6.96";
+    public static final String USER_SYST_NAME_ATT = "SNOMED_CT";
```

Comment:
Can you follow the same naming conventions as the other variables, namely change "ATT" to "ATTR"?